api/src/test/java/org/apache/iceberg/AssertHelpers.java

```diff
@@ -20,7 +20,9 @@
 package org.apache.iceberg;
 
 import java.util.concurrent.Callable;
-import org.junit.Assert;
+import org.assertj.core.api.AbstractThrowableAssert;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
```

Comment:
Can you remove this static import as well?

Reply:
done